MapleServer2/Commands/Game/GMCommands.cs

```diff
@@ -147,9 +147,10 @@ public AttributeCommand()
             new Parameter<string>("equipSlot", "Equip slot, e.g.: RH (ItemSlot.cs)"),
             new Parameter<string>("newAttributeId", "New Attribute, e.g.: Dex (StatAttribute.cs)"),
             new Parameter<float>("value", "Value, e.g.: 10 / 0.002"),
-            new Parameter<byte>("isPercentage", "Is percentage, e.g.: 1 / 0")
+            new Parameter<byte>("isPercentage", "Is percentage, e.g.: 1 / 0"),
+            new Parameter<byte>("category", "Stat Category, e.g.: 1 - 4"),
```

Comment:
Explain a little better that category is Constants, Randoms and statics. and the code uses 0 ~ 2 not 1 ~ 4